dxr/vcs.py

```diff
@@ -147,7 +147,7 @@ def generate_diff(self, path):
         return self.upstream + 'diff/' + self.previous_revisions[path] + '/' + path
 
     def generate_blame(self, path):
-        return self.upstream + 'annotate/' + self.revision + '/' + path
+        return self.upstream + 'annotate/' + self.revision + '/' + path + '#l{{line}}'
```

Comment:
Probably makes sense to use string formatting here, instead of repeated concatenation.